web/themes/custom/cybersecurity/css/styles.css

```diff
@@ -470,57 +470,99 @@ button {
   border: 0;
 }
 
-.container {
-  max-width: 1024px;
-  margin: 0 auto;
-  padding: 2rem;
+.toolbar-menu li {
+  font-size: small;
 }
 
-/**
- * List styles
- */
-.list li {
-  display: list-item;
-  padding: 1px 0;
+.tabs {
+  padding: 2rem 0;
 }
 
-.list--bullet {
-  list-style-type: disc;
-  padding-left: 16px;
+.tabs li {
+  font-size: 1.6rem;
 }
 
-.list--inline li {
-  display: inline-block;
+.tabs li a {
+  padding: 1rem 2rem;
 }
 
-@media only screen and (min-width: 765px) {
-  .list--inline-md li {
-    display: inline-block;
-  }
+.tabs li a.is-active {
+  background-color: #e5e5ec;
```

Comment:
It's not clear to me whether the project uses CSS precompilation or plain vanilla CSS. If the former, hex values would ideally be better set as a variable in case of re-use throughout the wider project.

Reply:
I'm using SCSS which is being compiled to CSS. All of the colours are set as variables in the variables.scss file, the only hex colours in the SCSS are the ones used to create the gradient in the header as they are not used elsewhere, although I could maybe set up the gradient as a variable itself so that it can be used again easily if required.

Reply:
Configure the colour vars in tailwind
https://tailwindcss.com/docs/customizing-colors
And in general have a look at the tailwind config, you can change most things to suit your theme.